core/chains/evm/txmgr/strategies_test.go

```diff
@@ -21,9 +21,9 @@ func Test_SendEveryStrategy(t *testing.T) {
 
 	assert.Equal(t, uuid.NullUUID{}, s.Subject())
 
-	n, err := s.PruneQueue(testutils.Context(t), nil)
+	ids, err := s.PruneQueue(testutils.Context(t), nil)
 	assert.NoError(t, err)
-	assert.Equal(t, int64(0), n)
+	assert.Equal(t, 0, len(ids))
```

Comment:
nit: think it might be cleaner to do something like: 
```suggestion
	assert.Len(t, ids, 0)
```